TESTS/dev_mgmt/update/main.cpp

```diff
@@ -195,37 +168,32 @@ void spdmc_testsuite_update(void) {
 
     test_case_finish("Connect to " TEST_NETWORK_TYPE, iteration + (net_status == 0), (net_status != 0));
 
-    if (iteration == 0) {
-        test_case_start("Format " TEST_FILESYSTEM_TYPE, 3);
-        logger("[INFO] Resetting storage to a clean state for test.\n");
-
-        int storage_status = fs.reformat(&sd);
-        if (storage_status != 0) {
-            storage_status = sd.erase(0, sd.size());
-            if (storage_status == 0) {
-                storage_status = fs.format(&sd);
-                if (storage_status != 0) {
-                    logger("[ERROR] Filesystem init failed\n");
-                }
-            }
-        }
+    test_case_start("Initialize Storage", 2);
 
-        // Report status to console.
-        if (storage_status == 0) {
-            logger("[INFO] Storage format successful.\r\n");
-        } else {
-            logger("[ERROR] Storage format failed.\r\n");
-            test_failed();
-        }
+    logger("[INFO] Attempting to initialize storage.\r\n");
+
+    // Default storage definition.
+    BlockDevice* bd = BlockDevice::get_default_instance();
+    FileSystem *fs = FileSystem::get_default_instance();
```

Comment:
Same comment for `connect` test - the function is inconsistent. The test should be able to decide on filesystem, similar to how a customer might decide to use LittleFileSystem as default for SD card, despite what this function returns.

Reply:
why do you need type of file system here?
in case this would be other than FAT , it will return it as default

Reply:
you are mostly using it to format SD, but we can't format it now before init
only after init part, the partitions are ready
this is the way factory configuration client works , have to be aligned with it